src/modules/message/dto/SendMessageRequestDto.ts

```diff
@@ -1,4 +1,11 @@
+import { IsString, IsNotEmpty } from 'class-validator';
+
 export class SendMessageRequestDto {
+  @IsString()
+  @IsNotEmpty()
```

Comment:
```suggestion
  @IsNotEmpty()
  @IsString()
```

Coloca nessa ordem, as verificações são executadas em ordem de precedência